staff/eric-bio.md

```diff
@@ -0,0 +1,9 @@
+Eric Herman is the Foundation for Public Code's Lead Codebase Steward for Quality. With two decades of experience, he brings the perspectives of codebases of many languages and development practices.
+
+He worked as a developer for MySQL and remains active in the Free and Open Source Software database community. He continues to contribute to MySQL and MariaDB. Eric serves as the chairman of the board of directors of the MariaDB Foundation.
+
+Formerly a Principal Developer at Booking<span></span>.com, he has worked on the data infrastructure of one of the top 10 largest ecommerce sites. During his time at Booking<span></span>.com, he developed performance enhancements to the Perl programming language which were contributed upstream for all users of Perl to benefit from.
```

Comment:
Remove the HTML `<span></span>` from this sentence.

Reply:
Removing the span will link the employer ... twice ... which is not what we want. Do you have an alternative way to write it such that it is not linked?

Reply:
addressed with commit caf364a09d38e3d12abe7c779be312e7bb9a39ac

>    Remove "example<span></span>.com" avoid-link-trick
>    
>    With Jekyll, the "span trick" is not needed to avoid the link.
>    It does not matter if the repo view version links, we only care about
>    the html rendered output.